File: Python WorldSimulatorGame/Wilk.py
from Zwierze import Zwierze


class Wilk(Zwierze):

    def __init__(self, polozenie, swiat):
        super(Wilk, self).__init__(9, 5, 'W', 0, True, polozenie, swiat)

    def getNazwaOrganizmu(self):
        return "Wilk"

File: C++ WorldSimulatorGame/POproj1/Wilk.cpp
#include "Wilk.h"

Wilk::Wilk(Punkt polozenie, Swiat* swiat)
	:Zwierze(9, 5, 'W', 0 , true, Organizmy::wilk, polozenie, swiat) {
	this->sila = 9;
	this->inicjatywa = 5;
	this->znak = 'W';
	this->wiek = 0;
}
string Wilk::getNazwaOrganizmu() {
	return "Wilk";
}